File: Easy/palindrome_number.java
class Solution {
    public boolean isPalindrome(int x) {
        String strX = String.valueOf(x);
        String strXR = new StringBuilder(strX).reverse().toString();
        return strX.equals(strXR);
    }
}

public class palindrome_number {
    public static void main(String[] args) {
        Solution solution = new Solution();
        System.out.println(solution.isPalindrome(121)); // true
        System.out.println(solution.isPalindrome(-121)); // false
        System.out.println(solution.isPalindrome(10)); // false
    }
}

File: Easy/palindrome_number.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function (x) {
  let strX = String(x);
  return strX === strX.split("").reverse().join("");
};

console.log(isPalindrome(121)); //true
console.log(isPalindrome(-121)); //false
console.log(isPalindrome(10)); //false
